service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -38,7 +38,8 @@
 import java.util.concurrent.Executors;
 
 /**
- * Use the RegistryManager class to manage the identity registry in IoT Hubs.
+ * Use the RegistryManager client to manage the identity registry in IoT hubs.
```

Comment:
I'd have preferred all our clients end with the suffix "Client", but we can at least refer to them as clients.